example/spec/parallel/ingest/ingestPdrWithNodeNameSpec.js

```diff
@@ -277,6 +277,9 @@ describe('Ingesting from PDR', () => {
           expectedParsePdrOutput.granules[0].dataType += testSuffix;
           expectedParsePdrOutput.pdr.name = pdrFilename;
           expectedParsePdrOutput.granules[0].provider = nodeNameProviderId;
+          expectedParsePdrOutput.granules.forEach((granule) => {
```

Comment:
These updates were needed to account for the new producerGranuleId showing up no matter what the configuration is.